translations/monero-core_fr.ts

```diff
@@ -1286,7 +1286,7 @@ Description : </translation>
     <message>
         <location filename="../main.qml" line="731"/>
         <source>Please wait...</source>
-        <translation>Veuillez patienter...</translation>
+        <translation>Veuillez patienter…</translation>
```

Comment:
Ditto. According to Unicode, ... and … convey the same exact meaning, so you can change it back if you're not comfortable with this, though. Still, I consider ellipsis to carry a semantic value dot dot dot doesn't have.